### DISCUSSION TITLE
ngMocksUniverse.global

### DISCUSSION THREAD

**Discussion by cheng93** [help-me-mom/ng-mocks#7842] [General]
Hi

I'm wondering why we don't remove the ngModule definition from `ngMocksUniverse.global`. I believe it's causing me some memory issues.

It looks to be set here

https://github.com/help-me-mom/ng-mocks/blob/master/libs/ng-mocks/src/lib/mock-builder/mock-builder.performance.ts#L59

![image](https://github.com/help-me-mom/ng-mocks/assets/13205912/af5bf309-1770-4efd-881c-35888c61a167)

![image](https://github.com/help-me-mom/ng-mocks/assets/13205912/aa369aa4-ee74-4d46-bd9b-f9f18c574224)

**Comment by satanTime** [comment]
Hi @cheng93, thanks for spotting this. 

I've create an issue to investigate how it can be fixed: https://github.com/help-me-mom/ng-mocks/issues/7843
